### ISSUE TITLE
FetchedValue can't be internal

### ISSUE THREAD

**Issue by andimarek** [graphql-java/graphql-java#2116]
`InstrumentationFieldCompleteParameters.fetchedValue` is actually `FetchedValue` and not `Object`. We should make it clear and therefore make `FetchedValue` public API.

**Comment by bbakerman** [comment]
actually its not quite true

```
    protected FieldValueInfo completeValueForList(ExecutionContext executionContext, ExecutionStrategyParameters parameters, Iterable<Object> iterableValues) {

        OptionalInt size = FpKit.toSize(iterableValues);
        ExecutionStepInfo executionStepInfo = parameters.getExecutionStepInfo();

        InstrumentationFieldCompleteParameters instrumentationParams = new InstrumentationFieldCompleteParameters(executionContext, parameters, () -> executionStepInfo, iterableValues);


```

In this case we have an iterable of values

**Comment by andimarek** [comment]
Addressed by #2690
